packages/cli/src/projects.test.ts

```diff
@@ -0,0 +1,252 @@
+import { MedplumClient } from '@medplum/core';
```

Comment:
Nice, I like breaking this out into separate file.

Any objections to using `project.ts` and `project.test.ts`?  We can establish the convention that the filename should === the command category.

Reply:
Yeah good cal, I'll change that